examples/index.html

```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
 
   <footer>
     <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
+    <div class="row">
```

Comment:
I suggest that the footer section should be at the  bottom of the page, I mean there should be no space below it as well.
Can you please do this after a review by @jywarren ?
Thanks